packages/gateway/src/config.ts

```diff
@@ -54,6 +54,8 @@ export async function loadConfig<
         !opts.quiet &&
           opts.log.info(`Found default config file ${absoluteConfigPath}`);
         const importUrl = pathToFileURL(absoluteConfigPath).toString();
+        !opts.quiet &&
+          opts.log.info(`Loading config file at path ${importUrl}`);
```

Comment:
There's already a similar enough log above, why have two?
```suggestion
```

Reply:
I wanted to see the import path which is created by `pathToFileURL` but I can remove it.